qubesappmenus/__init__.py

```diff
@@ -453,12 +461,26 @@ def _appmenus_create_onedir(self, vm, force=False, refresh_cache=True,
                                  os.environ.get('KDE_SESSION_VERSION', '4')])
 
     @staticmethod
-    def _is_old_path(name):
-        """Return if a path to a desktop name is for the old menu version.
+    def _is_very_old_path(name):
```

Comment:
Since both functions are used in the same place, better extend `_is_old_path`, instead of adding separate function.